SoulsFormats/SoulsFormats/Util/SFUtil.cs

```diff
@@ -519,14 +519,26 @@ public static void EncryptDS3Regulation(string path, BND4 bnd)
         public static BND4 DecryptERRegulation(string path)
         {
             byte[] bytes = File.ReadAllBytes(path);
+            if (BND4.IsRead(bytes, out BND4 bnd4)) return bnd4; //Delete this line if you want to use a separate method 
+            bytes = DecryptByteArray(erRegulationKey, bytes);
+            return BND4.Read(bytes);
+        }
+        
+        /// <summary>
+        /// Checks if the file is encrypted, and returns unencrypted ER Regulation file from the specified path.   
+        /// </summary>
+        public static BND4 GetUnencryptedERRegulation(string path) //delete this method if you want to just add the fix to DecryptERRegulation(string path)
+        {
+            byte[] bytes = File.ReadAllBytes(path);
+            if (BND4.IsRead(bytes, out BND4 bnd4)) return bnd4; 
```

Comment:
as well

Reply:
would you rather the fix be in the `DecryptERRegulation` or make a new method for each game, like `GetUnencryptedERRegulation`?